include/motor-control/core/tasks/tmc2130_motor_driver_task.hpp

```diff
@@ -81,11 +81,23 @@ class MotorDriverMessageHandler {
         LOG("Received read motor driver request: addr=%d", m.reg_address);
         uint32_t data = 0;
         if (tmc2130::registers::is_valid_address(m.reg_address)) {
-            driver.read(tmc2130::registers::Registers(m.reg_address), data,
-                        m.message_index);
+            auto converted_addr = static_cast<uint8_t>(m.reg_address);
+            uint32_t token = spi::utils::build_token(converted_addr);
+            driver.read(token, data, m.message_index);
```

Comment:
same as the 2160 with the `read()` call - this should still take a register address and also take a byte of tags or something